try:
    print(1)
except Exception as result:
    print(result)
else:
    print("没有异常是执行的代码")